config.inc.php

```diff
@@ -8,12 +8,26 @@
 $hosts = isset($_ENV['MYSQL_HOST']) ? $_ENV['MYSQL_HOST'] : 'mysql';
 foreach (explode(',', $hosts) as $index => $host) {
 	$config = &$cfg['Servers'][$index + 1];
-	$host   = trim($host);
+
+	// split into host[:port], user and pass
+	list($host, $user, $password) = explode(';', $host, 3) + [1 => '', 2 => ''];
```

Comment:
`trim()` for `$host` is still needed here, not below.
Maybe just `array_map('trim', ...)` everything?